.jshintrc

```diff
@@ -1,3 +1,8 @@
 {
+<<<<<<< HEAD
```

Comment:
it is needed to resolve the conflicts by removing this kind of lines

Reply:
Fixed